hw/ip/pattgen/doc/dv/index.md

```diff
@@ -8,19 +8,27 @@ title: "PATTGEN DV document"
   * Develop and run all tests based on the [testplan](#testplan) below towards closing code and functional coverage on the IP and all of its sub-modules
 * **FPV**
   * Verify TileLink device protocol compliance with an SVA based testbench
+  * Build comprehensive coverage framework to measure coverage provided by existing tests
 
 ## Current status
 * [Design & verification stage]({{< relref "hw" >}})
   * [HW development stages]({{< relref "doc/project/development_stages.md" >}})
+  * Documentation is complete
+  * Testbench is done with coverage and test siute in development
+  * All tests are passing
```

Comment:
These are included in `HW development stages` https://docs.opentitan.org/hw/ip/otp_ctrl/doc/checklist/index.html#v1  so I think you do not need to rewrite them here :)